model/clusters_mgmt/v1/add_on_requirement_type.model

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Representation of an add-on requirement.
+class AddOnRequirement {
+
+	// Link to add-on.
+	link Addon AddOn
```

Comment:
I think we should start to avoid these "back links". Then aren't really useful and they introduce circular references that are harmless now but can block evolution of the architecture in the future. So please remove it.